source/client-side-encryption/tests/legacy/kmipKMS.yml

```diff
@@ -4,8 +4,34 @@ database_name: &database_name "default"
 collection_name: &collection_name "default"
 
```

Comment:
The legacy tests are generated by templates. Suggest adding a kmipKMS.yml.template. That may resolve the failing GitHub action.